scripts/e2e.sh

```diff
@@ -12,21 +12,23 @@ export TRAVIS=true
 export DBUS_SESSION_BUS_ADDRESS=/dev/null
 
 export TEST_SUITE_TYPE="junit"
-export TEST_RESULT_FILE_DIR="${REPO}/build2/e2e"
+export TEST_RESULT_FILE_DIR="${REPO}/build2/reports"
 
 export ISSUER=https://samples-javascript.okta.com/oauth2/default
 export CLIENT_ID=0oapmwm72082GXal14x6
 export USERNAME=george@acme.com
 get_secret prod/okta-sdk-vars/password PASSWORD
+export DEFAULT_TIMEOUT_INTERVAL=45000
 
 cd ${OKTA_HOME}/${REPO}
 
 function run_tests() {
-    exit 0;
-    # npm run pretest
-    # npm run test:okta-hosted-login
-    # kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN)
-    # npm run test:custom-login
+    # exit 0;
+    npm run pretest
+    npm run test:okta-hosted-login
+    kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN)
+    kill -s TERM $(lsof -t -i:8000 -sTCP:LISTEN)
```

Comment:
Why is this here twice?

Reply:
@swiftone 
There are different ports, 8080 and 8000 :)

Reply:
I am blind